catalogue/webapp/components/VisuallySimilarImagesFromApi/VisuallySimilarImagesFromApi.tsx

```diff
@@ -52,7 +52,7 @@ const VisuallySimilarImagesFromApi: FunctionComponent<Props> = ({
       <h3 className={font('wb', 5)}>Visually similar images</h3>
       <Wrapper>
         {similarImages.map(related => (
-          <a key={related.id} onClick={() => onClickImage(related)} href="#">
+          <a key={related.id} onClick={() => onClickImage(related)}>
```

Comment:
I think removing the href, will remove the ability to tab to the link. Does make me question whether this should be a link - but that is perhaps one for another day.

Reply:
You're right. I've pushed a new commit with a suggestion where it's still a link, but at least it follows the model I've done for the Results images, what do you think?